api.py

```
@@ -277,3 +297,46 @@ async def api_athlete_pb_table(
                 '<script>window.location.href="https://youtu.be/dQw4w9WgXcQ?si=JPPysw3QXTLBs71z"; window.location.reload()</script>'
             )
+
+
+@router.get(
+    "/athletes",
+    response_class=JSONResponse,
+    summary="Returns json data of athletes in the database",
+    description="TODO: WIP"
+)
+async def api_athletes_endpoint(
+    request: Request,
```

Ruff (0.14.5)

Unused function argument: `request`

(ARG001)